bin/node/runtime/src/lib.rs

```diff
@@ -1847,6 +1846,7 @@ impl_runtime_apis! {
 			// which is why we need these two lines below.
 			use pallet_session_benchmarking::Pallet as SessionBench;
 			use pallet_offences_benchmarking::Pallet as OffencesBench;
+			use pallet_election_provider_support_benchmarking::Pallet as PepsoBench;
```

Comment:
we can drop the the `P`, signifying `Pallet`, and call this `EPSBench`: Election provider support.